actor/src/main/scala/org/apache/pekko/actor/Scheduler.scala

```diff
@@ -75,8 +75,8 @@ trait Scheduler {
    * If the `Runnable` throws an exception the repeated scheduling is aborted,
    * i.e. the function will not be invoked any more.
    *
-   * @throws java.lang.IllegalArgumentException if the given delays exceed the maximum
-   * reach (calculated as: `delay / tickNanos > Int.MaxValue`).
+   * @throws java.lang.IllegalArgumentException if the given delays is zero, negative or exceed the maximum
+   * reach (calculated as: `delay / tickNanos > Int.MaxValue`)
```

Comment:
can you undo this change?

Reply:
what's wrong, i thinkg it is good to have a clarity javadoc.

Reply:
It's the `.` that was removed - can you add that back?